src/lib.rs

```diff
@@ -189,14 +228,26 @@ impl Drop for SvgImage {
  * Loads SVG data from a file at the given `Path`.
  *
  * # Arguments
- * - `svg_path` - Path to the SVG you want to load
+ * - `svg_path` - Text contents of the SVG you want to load
```

Comment:
I think these comments got switched around by accident. Shouldn't this remain as "Path to the SVG you want to load"?

Then the documentation for `parse_str` seems like it should be "Text contents of the SVG you want to load".

Reply:
Yep, looks like I copy-pasted at the wrong location. I'll get that updated. Thanks!